### PR TITLE
CoPa: #2985, do not toggle LP, but set explicitely

### PR DESCRIPTION
**Pull request by nhaagen**
https://mantis.ilias.de/view.php?id=29859

### CODE REVIEW

File: Modules/ContentPage/classes/class.ilContentPageKioskModeView.php [hunk 1]
```diff
@@ -124,20 +127,26 @@ public function updateGet(State $state, string $command, int $param = null) : St
...
+                $marks->setCompleted($new_state);
+                $marks->update();
                 ilLPStatusWrapper::_updateStatus($this->contentPageObject->getId(), $this->user->getId());
 
-                $this->lng->loadLanguageModule('trac');
-
-                $this->messages[] = $this->uiFactory->messageBox()->success(
-                    $this->lng->txt('trac_updated_status')
-                );
+                if($old_state != $new_state) {
```
Comment (mjansenDatabay) [inline Modules/ContentPage/classes/class.ilContentPageKioskModeView.php]: @nhaagen A space is missing here (PSR 2 + x coding style ;))

**Review by mjansenDatabay — CHANGES_REQUESTED** [review]

**Comment by mjansenDatabay** [comment]
Thx a lot @nhaagen , I only added a remark regarding the coding style.

**Comment by klees** [comment]
@nhaagen I have a pre-commit hook for this, I‘ll share soon.

**Comment by mjansenDatabay** [comment]
> @nhaagen I have a pre-commit hook for this, I‘ll share soon.

Or you can use `CaptainHook` at least for `trunk` and `release_7`.

**Review by mjansenDatabay — APPROVED** [review]

**Pull request merged**